### ISSUE TITLE
keyboard access - skip to main content 

### ISSUE THREAD

**Issue by SteveM-aus** [civictheme/uikit#319]
### Summary

The side navigation includes a skip to main content feature available by keyboard access; unfortunately when this is accessed by keyboard and the user is taken to the main content, when they try to tab forward from there they are returned to the skip to main content feature on the side navigation. 

### Steps to reproduce

tab through the site from the tip of the page
upon reaching the left hand navigation "skip to main content" presents
hit enter to interact with the "skip to main content" link
you are at the top of the main content
when tabbing on form this position, you are automatically returned to the left hand navigation "skip to main content"

### Observed outcome

This results in the user potentially being trapped in a loop and having to find an alternative (unintuitive) method to progress through the content.

 This is related to WCAG 2.2 criteria such as logical order 2.4.3 and Keyboard access 2.1.1 and 2.1.2 

### Expected outcome

A keyboard user should be able to select skip to main content and continue tabbing through the site from there

**Comment by fionamorrison23** [comment]
@jcloys can you please test this and let me know the results? Thanks.

**Comment by fionamorrison23** [comment]
Noting that upon testing we got different results on different CivicTheme sites. On civictheme.io upon hitting enter on the 'Skip to main content', upon next Tab the button in the banner is focussed. This appears to be expected behaviour.

**Comment by richardgaunt** [comment]
@SteveM-aus Current 1.8 CivicTheme behaviour is first tab to "Skip to main content", then if this link is activated you are taken just above the `h1` in the banner and you tab on from there.

You don't go through the main menu.

**Comment by fionamorrison23** [comment]
@richardgaunt in this case the reporter is referring to the Skip to main content link in the side nav not the top of the page.
However, I have checked this and I can't reproduce this issue. For me, once I hit Skip in the side nav I go to the main content, it then tabs through the main content and then takes me back to the side nav. 

@jcloys can you please have a look at this, is this now showing the expected behaviour? I think there are still some other keyboard issues to sort out, but for this specific issue I'd like to close it once you have reviewed. Thanks.

**Comment by richardgaunt** [comment]
I believe we fixed this a few months back

**Comment by fionamorrison23** [comment]
Closing as this is fixed.
